src/Collection.php

```diff
@@ -65,6 +65,73 @@ public function __construct(Manager $manager, $ns, WriteConcern $wc = null, Read
     }
 
     /**
+     * Create a single index in the collection.
+     *
+     * @see http://docs.mongodb.org/manual/reference/command/createIndexes/
+     * @see http://docs.mongodb.org/manual/reference/method/db.collection.createIndex/
+     * @param array|object $keys
+     * @param array        $options
+     * @return string The name of the created index
+     */
+    public function createIndex($keys, array $options = array())
```

Comment:
The user should be able to specify a BSON object.

I was thinking of the case where they want to index the `0` field ascending. We shouldn't send over a BSON array. While I could force-cast to an object internally, I figure this leaves them the flexibility and also means we can take a BSON object (whenever we create that type) here.